spec/data_migrations/permissions_spec.rb

```diff
@@ -67,6 +67,38 @@
       end
     end
 
+    describe 'update permissions for super admin role to be able to force publish' do
+      let(:given_task_name) {':hbx_admin_can_force_publish'}
+
+      before do
```

Comment:
Let's move each one of these instance variables to it's own `let` within it's own `context` and then separate each `expect`ation to it's own `it` block.

Reply:
made changes as suggested, Thanks Bill.